normandy/recipes/api/views.py

```diff
@@ -96,6 +100,14 @@ def update(self, request, *args, **kwargs):
 
         return super().update(request, *args, **kwargs)
 
+    @detail_route(methods=['GET',])
+    def history(self, request, pk=None):
+        recipe = self.get_object()
+        content_type = ContentType.objects.get_for_model(recipe)
+        versions = Version.objects.filter(content_type=content_type, object_id=recipe.pk)
+        data = [RecipeVersionSerializer(v, context={'request': request}).data for v in versions]
```

Comment:
Better would be `data = RecipeVersionSerializer(versions, context={'request': request}, many=True)`. Generally if you have to access `.data` on more than one serializer in a request, something is a little fishy.